geniza/corpus/templates/corpus/document_list.html

```diff
@@ -11,7 +11,11 @@ <h1 class="sr-only">{{ page_title }}</h1>
             {{ form.q }}
             {{ form.sort }}
             <label for="sort">{{ form.sort.label }}</label>
-            <input type="submit" value="search" />
+            {# Translators: Search submit button #}
+            {% translate 'Submit search' as search_label %}
+            <button type="submit" aria-label="{{ search_label }}">
+                <i class="ph-magnifying-glass"></i>
+            </button>
```

Comment:
I was testing a little bit with both screen reader and keyboard navigation; the tab order is confusing since it doesn't match the visual layout — although of course it makes sense to have a submit button after the end of the form so you can submit any changes. I'm not requesting any changes now, but I think we'll need to add that second submit button at the bottom of the form sooner then later, and we should move this one so it's position in the DOM is consistent with where it's displayed.

Reply:
Yes, I wanted at least the DOM to have the submit button at the end of the form, since it might be confusing for screen readers to encounter the submit button before some form fields. But the tradeoff is that it does make keyboard navigation somewhat confusing. I wonder how most people handle this—it doesn't seem like WCAG at least requires you to put the submit button at the end of the form, so maybe it could just exist in the middle of it even in the DOM.